support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -222,7 +227,7 @@ PhaseChangeAndTriggers:
           Offset: 0
     PhaseChanges:
       - CheckpointAndExitAfterWallclock:
-          WallclockHours: 23.5
+          WallclockHours: 47.5
```

Comment:
Keep this at 23.5 hours please. We'll have to adjust it on clusters with longer time limits, but we have to do that more dynamically than this.

Reply:
This too - it was from one of my test runs.